### DISCUSSION TITLE
GDAS DATA ERROR

### DISCUSSION THREAD

**Discussion by vibinj** [NASA-LIS/LISF#1399] [Q&A]
Hi,
I have downloaded the GDAS data for the year 2020. While running the LIS, I am facing an error "[INFO] Reading GDAS file1 (A)
 /projects/iist01/vibin/GDAS/202001/2020010118.gdas1.sfluxgrbf06.sg  [ERR] Could not retrieve entries in file:"  I have tried with 2012 GDAS data it is working without any error.

Please help

**Comment by dmocko** [comment]
Hi @vibinj 

It's possible that this file was corrupted during your download.  Can you try to download it again please?

https://portal.nccs.nasa.gov/lisdata_pub/data/MET_FORCING/GDAS/202001/

The file size is 188317198 bytes.
